collectors/ebpf.plugin/README.md

```diff
@@ -100,6 +100,8 @@ accepts the following values:
     charts for the return of these functions, such as errors. Monitoring function returns can help in debugging software,
     such as failing to close file descriptors or creating zombie processes.
 -   `update every`:  Number of seconds used for eBPF to send data for Netdata.
+-   `update intragtion every`:  Number of seconds used for eBPF to sync data with `apps.plugin` and `cgroup.plugin`. The
+     default and acceptable minimum value is `10` seconds.
```

Comment:
Do we need this option? When/why do you need to increase it?

Reply:
It is not mandatory, I added it to allow user to reduce even more the frequency that we parse `/proc` to keep compatibility with `apps.plugin`. In the ideal world we will change `apps` and `ebpf` to have a shared memory between them, like we have right now between `ebpf.plugin` and `cgroup.plugin` (ping @cpipilas , @stelfrag ).

Reply:
>  I added it to allow user to reduce even more
> to keep compatibility with apps.plugin

What is that compatibility? Why _reduce even more_ to keep compatibility? When do you need to do it?

Reply:
Sorry, it is not `reduce even more`, it is extend, because larger period of time means less process usage.

We were doing in the same frequency than `apps` by default, I am extending this, because we do not have one unique thread sending data every second by default.

Reply:
I'm on Ilya's side. I would say that if you don't allow it to be less than 10 sec., you shouldn't add the option at all. But, if you allow it to be 1-10, then it makes sense.

Reply:
Considering that we will add shared memory between two plugins soon, I removed the option with latest commit.